seed_stage_based_messaging/testsettings.py

```diff
@@ -26,3 +26,19 @@
 METRICS_AUTH_TOKEN = "REPLACEME"
 
 PASSWORD_HASHERS = ('django.contrib.auth.hashers.MD5PasswordHasher',)
+
+
+# REST Framework conf defaults
+REST_FRAMEWORK = {
+    'PAGE_SIZE': 2,
+    'DEFAULT_PAGINATION_CLASS':
+        'rest_framework.pagination.CursorPagination',
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework.authentication.BasicAuthentication',
+        'rest_framework.authentication.TokenAuthentication',
+    ),
+    'DEFAULT_PERMISSION_CLASSES': (
+        'rest_framework.permissions.IsAuthenticated',
+    ),
+    'DEFAULT_FILTER_BACKENDS': ('rest_framework.filters.DjangoFilterBackend',)
+}
```

Comment:
My only concern is that here it might become easy for example for the `DEFAULT_AUTHENTICATION_CLASSES` to become out of sync between test and production and we might introduce a bug. Is it possible to just override the thing we care about for the purposes of the test (the `PAGE_SIZE`)?